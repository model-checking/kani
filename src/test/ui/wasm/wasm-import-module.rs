#[link(name = "...", wasm_import_module)] //~ ERROR: must be of the form
extern "C" {}

#[link(name = "...", wasm_import_module(x))] //~ ERROR: must be of the form
extern "C" {}

#[link(name = "...", wasm_import_module())] //~ ERROR: must be of the form
extern "C" {}

fn main() {}
