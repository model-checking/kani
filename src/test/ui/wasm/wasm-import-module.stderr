error: must be of the form `#[link(wasm_import_module = "...")]`
  --> $DIR/wasm-import-module.rs:1:22
   |
LL | #[link(name = "...", wasm_import_module)]
   |                      ^^^^^^^^^^^^^^^^^^

error: must be of the form `#[link(wasm_import_module = "...")]`
  --> $DIR/wasm-import-module.rs:4:22
   |
LL | #[link(name = "...", wasm_import_module(x))]
   |                      ^^^^^^^^^^^^^^^^^^^^^

error: must be of the form `#[link(wasm_import_module = "...")]`
  --> $DIR/wasm-import-module.rs:7:22
   |
LL | #[link(name = "...", wasm_import_module())]
   |                      ^^^^^^^^^^^^^^^^^^^^

error: aborting due to 3 previous errors

